playwright-e2e/tests/rgp/adult-enrollment.spec.ts

```diff
@@ -0,0 +1,158 @@
+import { expect, Page } from '@playwright/test';
+import * as auth from 'authentication/auth-rgp';
+import * as user from 'data/fake-user.json';
+import { APP } from 'data/constants';
+import { test } from 'fixtures/rgp-fixture';
+import DashboardPage from 'pages/rgp/dashboard-page';
+import HowItWorksPage from 'pages/rgp/how-it-works-page';
+import TellUsAboutYourFamilyPage from 'pages/rgp/enrollment/tell-us-about-your-family-page';
+import TellUsYourStoryPage, { WHO } from 'pages/rgp/enrollment/tell-us-your-story-page';
+import HomePage from 'pages/rgp/home-page';
+import { setAuth0UserEmailVerified } from 'utils/api-utils';
+
+const { RGP_USER_EMAIL, RGP_USER_PASSWORD } = process.env;
+
+/** UNFINISHED **/
+
+test.describe('Adult Self Enrollment', () => {
+  const assertProgressActiveItem = async (page: Page, itemName: string): Promise<void> => {
+    const locator = page.locator('li.activity-stepper__step-container button.stepper-btn.stepper-btn--active');
+    await expect(locator).toHaveCount(1);
+    await expect(locator).toContainText(itemName);
+  };
+
+  test('Can complete application @functional @enrollment @rgp', async ({ page }) => {
+    const homePage = new HomePage(page);
+    await homePage.clickGetStarted();
+
+    const howItWorks = new HowItWorksPage(page);
+    await howItWorks.startApplication();
+
+    const tellUsYourStoryPage = new TellUsYourStoryPage(page);
+    await tellUsYourStoryPage.waitForReady();
+    // Check all checkboxes
+    await tellUsYourStoryPage.who().check(WHO.UnderstandEnglishOrSpanish);
+    await tellUsYourStoryPage.who().check(WHO.LivesInUS);
+    await tellUsYourStoryPage.who().check(WHO.HasRareGeneticallyUndiagnosedCondition);
+    await tellUsYourStoryPage.who().check(WHO.IsUnderCare);
+    await tellUsYourStoryPage.submit();
+
+    const userEmail = await auth.createAccountWithEmailAlias(page, {
+      email: RGP_USER_EMAIL,
+      password: RGP_USER_PASSWORD,
+      waitForNavigation: false,
+      waitForAuth: false
+    });
+
+    await expect(page.locator('text="Account Verification"')).toBeVisible();
+    await expect(page.locator('#view__account-verification p:first-child')).toHaveText(
+      'An email has been sent to the address that you provided. ' +
+        'Please check your inbox, ' +
+        'open the email and click "Verify my account and complete questionnaire" ' +
+        'to move forward in the submission process.'
+    );
+    await expect(page.locator('#view__account-verification p:last-child')).toHaveText(
+      'If you do not receive an email, contact us at raregenomes@broadinstitute.org.'
+    );
+
+    // Verify user email by Auth0 API
+    await setAuth0UserEmailVerified(APP.RPG, userEmail, { isEmailVerified: true });
```

Comment:
do we want to fix the typo from `RPG` to `RGP`

Reply:
Good catch. I'll fix this in next PR since new commits will clear existing approvals.